### PR TITLE
Add Bayes factor to Benford's law and do some markup

### PR DESCRIPTION
**Pull request by koenderks**

### CODE REVIEW

File: R/auditClassicalNumberBunching.R [hunk 1]
```diff
@@ -113,20 +113,18 @@ auditClassicalNumberBunching <- function(jaspResults, dataset, options, ...){
...
-    table$addColumnInfo(name = 'avgFreq', title = gettext('Average'), 		type = 'number', overtitle = gettext('Frequency'))
-    table$addColumnInfo(name = 'pvalue1', title = gettext("<i>p</i> value"), 	type = 'pvalue', overtitle = gettext('Frequency'))
+    table$addColumnInfo(name = 'avgFreq', title = gettext('Average'), 	type = 'number', overtitle = gettext('Frequency'))
+    table$addColumnInfo(name = 'pvalue1', title = gettext("p"), 		type = 'pvalue', overtitle = gettext('Frequency'))
   }
   if(options[["entropy"]]){
-    table$addColumnInfo(name = 'entropy', title = gettext('<i>S</i>'), 		type = 'number', overtitle = gettext('Entropy'))
-    table$addColumnInfo(name = 'pvalue2', title = gettext("<i>p</i> value"), 	type = 'pvalue', overtitle = gettext('Entropy'))
+    table$addColumnInfo(name = 'entropy', title = gettext('S'), 		type = 'number', overtitle = gettext('Entropy'))
+    table$addColumnInfo(name = 'pvalue2', title = gettext("p"), 		type = 'pvalue', overtitle = gettext('Entropy'))
```
Comment (boutinb) [inline R/auditClassicalNumberBunching.R:124]: also these gettextf. I think you remove the gettext function

File: R/auditClassicalNumberBunching.R [hunk 2]
```diff
@@ -229,14 +227,14 @@ auditClassicalNumberBunching <- function(jaspResults, dataset, options, ...){
...
-    table$addColumnInfo(name = 'df', 		title = gettext('df'), 			   type = 'integer')
-    table$addColumnInfo(name = 'pvalue',	title = gettext("<i>p</i> value"), type = 'pvalue')
+    table$addColumnInfo(name = 'var1', 		title = "", 			type = 'string')
+    table$addColumnInfo(name = 'separator', title = "", 			type = 'separator')
+    table$addColumnInfo(name = 'var2', 		title = "", 			type = 'string')
+    table$addColumnInfo(name = 'n', 		title = gettext("n"), 	type = 'integer')
+    table$addColumnInfo(name = 'cor', 		title = gettext('r'), 	type = 'number')
+    table$addColumnInfo(name = 't', 		title = gettext('t'), 	type = 'number')
+    table$addColumnInfo(name = 'df', 		title = gettext('df'), 	type = 'integer')
+    table$addColumnInfo(name = 'pvalue',	title = gettext("p"), 	type = 'pvalue')
```
Comment (boutinb) [inline R/auditClassicalNumberBunching.R:237]: Same here

**Review by boutinb — CHANGES_REQUESTED** [review]
This PR could be a good testcase for https://github.com/jasp-stats/jasp-actions/pull/3.
So if you are not in a hurry, let's wait that this action is implemented.

**Comment by koenderks** [comment]
To be honest, I would rather have it in the module now

**Review by boutinb — APPROVED** [review]

**Pull request merged**